Add missing unitary authorities (#65)

joescottdave opened the pull request:
- fix: Add North and West Northamptonshire to counties #65
- fix: Add Bournemouth, Christchurch and Poole to counties
- fix: Add Somerset West and Taunton to districts
- fix: Add East Suffolk to districts
- fix: Add West Suffolk to districts

joescottdave committed:
fix: Add North and West Northamptonshire to names

joescottdave committed:
fix: Add North and West Northamptonshire to names

joescottdave committed:
Merge branch '65-missing-new-counties' of github.com:epimorphics/standard-reports-ui into 65-missing-new-counties

joescottdave committed:
Merge branch '65-missing-new-counties' of github.com:epimorphics/standard-reports-ui into 65-missing-new-counties

joescottdave committed:
Merge branch '65-missing-new-counties' of github.com:epimorphics/standard-reports-ui into 65-missing-new-counties

joescottdave committed:
Merge branch '65-missing-new-counties' of github.com:epimorphics/standard-reports-ui into 65-missing-new-counties

joescottdave committed:
Merge branch '65-missing-new-counties' of github.com:epimorphics/standard-reports-ui into 65-missing-new-counties

joescottdave committed:
Merge branch '65-missing-new-counties' of github.com:epimorphics/standard-reports-ui into 65-missing-new-counties

joescottdave committed:
Merge branch '65-missing-new-counties' of github.com:epimorphics/standard-reports-ui into 65-missing-new-counties

ijdickinson reviewed (commented):
We should check the other UAs that have changed as well (I don't remember making edits for these changes): 

https://en.wikipedia.org/wiki/2019%E2%80%932023_structural_changes_to_local_government_in_England

joescottdave commented:
> 
> 
> We should check the other UAs that have changed as well (I don't remember making edits for these changes):
> 
> https://en.wikipedia.org/wiki/2019%E2%80%932023_structural_changes_to_local_government_in_England

Seems that there is one other new unitary authority **Bournemouth, Christchurch and Poole** and an additional three districts: 

1. Somerset West and Taunton
2. West Suffolk
3. East Suffolk

I'm assuming that Dorset (now Dorset Council) and Buckinghamshire (County Council, now simply Council) having merged with other authorities but retained their names won't need changing

joescottdave committed:
feat: Add additional new local authorities

Includes the new unitary authority of 'Bournemouth, Christchurch and Poole', and three new districts: 'Somerset West and Taunton', 'West Suffolk', and 'East Suffolk'

joescottdave committed:
docs: Note additional counties in changelog

ijdickinson reviewed (approved):
Lgtm. 

We should probably get the product owner to also :+1: on preprod before going live. (Or Mal, but I don't think Mal has much availability at the moment)
